network/p2p/ping.go

```diff
@@ -16,9 +16,6 @@ import (
 	"github.com/onflow/flow-go/network/message"
 )
 
-// the Flow Ping protocol prefix
-const FlowLibP2PPingPrefix = "/flow/ping/"
```

Comment:
moved to `libp2pNode.go` where the other protocol ID is defined to have both in the same place